inc/lightbox/self_test.php

```diff
@@ -52,7 +52,7 @@
             <?php if ( function_exists( 'ftp_connect' ) ): ?>
             <span style="background-color: #33cc33">Installed</span>
             <?php else: ?>
-            <span style="background-color: #FFFF00">Not installed</span>
+            <span style="background-color: #FF0000">Not installed</span>
```

Comment:
Most environments do not use FTP.  This item should be marked yellow (`#FFFF00`) and "Not detected".  It is similar to the SSH2 item below, which is already yellow and "Not detected".

Reply:
changed to yellow not detected